tests/netebpfext_unit/netebpfext_unit.cpp

```diff
@@ -373,11 +385,13 @@ netebpfext_unit_invoke_sock_addr_program(
         REQUIRE(sock_addr_context->msg_src_port == htons(5678));
     }
 
-    // If the action is round robin, decide the action based on the port number.
-    if (client_context->sock_addr_action == SOCK_ADDR_TEST_ACTION_ROUND_ROBIN) {
-        action = _get_sock_addr_action(sock_addr_context->user_port);
-    } else {
-        action = client_context->sock_addr_action;
+    if (is_admin) {
```

Comment:
Since the UM implementation of `SeAccessCheckFromState` currently always returns true, I think instead changing the return value of this function depending on the return value, we can simply add a `REQUIRE` check that the helper function always returns 1

Reply:
1. UM SeAccessCheckFromState returns false, with the fault injection code added.
2. REQUIRE cannot be applied in ebpf program.
The reason is when REQUIRE condition is not met, it takes to helper's destructor where NMRDeregisterClient and NMRWait for completion are invoked, which is incorrect in user mode ebpf program test.